atest/testdata/config/06_override_params/project_config_02.json

```diff
@@ -29,7 +25,7 @@
    "[import]": "./common/definitions_common.json",
    "${gPreproString}": "Override string param in imported file.",
    "params":{
-     "global":{
+     "glob":{
```

Comment:
Hi  Son,
where is "glob" coming from?
I did not agree to this.
Thank you,
 Thomas

Reply:
Hello Thomas,

Due to the "global" is reserved keyword of Python, so some scenarios of pytest failed when I name the key of Json object is "global".
That the reason I just changed this key name of Json configuration file.

Thank you,
Son

Reply:
Hello Thomas,

I created [new commit ](https://github.com/test-fullautomation/python-jsonpreprocessor/pull/30/commits/5a7d154f3e29f39a894ef6682f2619a6801f00cc) to use object name 'global' instead of 'glob'.
I also run this selftest on both Windows and OSD6.

Thank you,
Son